packages/xo-server/src/xapi/index.js

```diff
@@ -415,6 +415,44 @@ export default class Xapi extends XapiBase {
     await this.call('host.enable', this.getObject(hostId).$ref)
   }
 
+  @deferrable
```

Comment:
You can use `@deferrable.onError(log.warn)` to automatically log all deferred's errors.